authentication/api/main.py

```diff
@@ -37,6 +37,10 @@ def get_authentication_credentials(
 ):
     """
     The resource server will use client credentials to connect to the introspection endpoint
+
+    This function is a placeholder. Authentication will be provided
+    by Perseus Directory issued client certificates in production, and necessary client
+    information will be available in the certificate.
```

Comment:
This will shortly be deprecated/not required (when directory issued mtls certs arrive), so I've made it clear this is a placeholder

Reply:
You could pop it in with a small bit of copy and paste...
https://github.com/icebreakerone/demo-energy-mtlsonly/blob/dbf8b841e8b741fb4c076f77195718813ead5944/resource/api/main.py#L61
https://github.com/icebreakerone/demo-energy-mtlsonly/blob/dbf8b841e8b741fb4c076f77195718813ead5944/resource/api/main.py#L72

Reply:
It's a good point! I wanted to wait until we had the directory ready before adding it though, as we don't currently have an easy way of issuing client certificates to users. What do you think?

Reply:
Just need a slightly different subject in the client certificate Subject. Eg:
https://github.com/icebreakerone/demo-energy-mtlsonly/blob/dbf8b841e8b741fb4c076f77195718813ead5944/scripts/certmaker.sh#L61

Reply:
I've added https://github.com/icebreakerone/perseus-demo-energy/issues/43 for mtls role checking